Evaluation/Input/evaluation_plan.json

```diff
@@ -0,0 +1,497 @@
+{
+    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.0/OSP_Qualification_Plan_Schema.json",
```

Comment:
"$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/**v2.0**/
why not 2.1?